app/controllers/advertisements_controller.rb

```diff
@@ -0,0 +1,28 @@
+class AdvertisementsController < ApplicationController
+  def index
+    @advertisement = Advertisement.all
+  end
+
+  def show
+    @advertisement = Advertisement.find(params[:id])
+  end
+
+  def new
+    @advertisement = Advertisement.new
+  end
+
+  def create
+    @advertisement = Advertisement.new
```

Comment:
talked to my mentor and he said because we used `render :new` we needed the instance variable